src/lib/utils.test.ts

```diff
@@ -132,6 +132,90 @@ describe('utils', () => {
         decodedValue: ['0xcafecafe', '11', '8'],
         encodedValue: '0xcafecafe000000000000000b00000008',
       },
+      {
+        schema: {
+          name: 'TupleMultiType',
+          key: '1e1bc4abe01b7baa7d4a359c0f460e632ef34b3f16f5722bd8892f2dae913022',
+          keyType: 'Singleton',
+          valueType: '(bytes4,address,bytes2)[CompactBytesArray]',
+          valueContent: '(Bytes4,Address,Bytes)',
```

Comment:
@JeneaVranceanu `Bytes` is allowed as a `valueContent`. There is not a direct 1 to 1 link between the types in `valueType` and the options provided in `valueContent`.

For instance, the 3rd value provided in the tuple `bytes2` could be interprated as:
- a `Number`
- a `String` (for some text).
- a `Bytes2`
- a `Bytes` (if wanted)

There is no strict rules and direct relations. It is for how it could be interpreted. Interpreting as `Bytes` for any `bytesN` as it is bytes at the end.

Reply:
Talked about the same with @b00ste just forgot to leave a comment here.
Totally makes sense, it was my mistake. Thanks for the clarifications.